packages/test-helpers/package.json

```diff
@@ -28,8 +28,8 @@
     "access": "public"
   },
   "dependencies": {
-    "@open-wc/testing": "^3.0.0-next.5",
-    "@types/mocha": "^5.2.7",
-    "mocha": "^6.2.3"
+    "@open-wc/testing": "^3.2.0",
+    "@types/mocha": "^9.1.1",
```

Comment:
`@types/mocha` should be devDep instead?

Reply:
I think it should remain in dependencies. We ship mocha and type as a dependencies to users (mainly us).